### PR TITLE
Jekyll

### PR DESCRIPTION
**Pull request by gauntface**
A Jekyll template for the project - WooHoo.

This is an altered version of the template from a new Jekyll project.

@surma fancy a cheeky review?

### CODE REVIEW

File: docs-template/_includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<footer class="site-footer">
+
+  <div class="wrapper">
+
+    <div class="footer-col-wrapper">
+      <div class="footer-col footer-col-1">
+        <ul class="contact-list">
+          <li><a href="{{ site.github.issues_url | replace: 'http://', '//' }}">Raise an issue</a></li>
```
Comment (surma) [inline docs-template/_includes/footer.html]: I’d go with @paulirish’s approach here and always use https

File: docs-template/_includes/head.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.github.project_title | escape }}{% endif %}</title>
+  <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.github.project_tagline }}{% endif %}">
+
+  <!-- The docs server over HTTPS, but site.github.url points to HTTP resouce causing mixed content -->
+  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: site.github.url | replace: 'http://', '//' }}">
+  <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.github.url | replace: 'http://', '//' }}">
+  <link rel="alternate" type="application/rss+xml" title="{{ site.github.project_title }}" href="{{ "/feed.xml" | prepend: site.github.url | replace: 'http://', '//' }}">
```
Comment (surma) [inline docs-template/_includes/head.html]: Enforce https here, too

**Comment by surma** [comment]
@gauntface LGTM. I assume all the HTML ’n stuff will be minified?

**Comment by gauntface** [comment]
I am largely hoping that will be done by Jekyll / github pages.

**Comment by surma** [comment]
Yeah no it won’t...

**Pull request merged**